google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -300,6 +302,11 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      // Called on New Project or New Module dialogs.
```

Comment:
By adding framework support, do you mean going to Project Settings and adding the GAE facet there? Or clicking the Framework discovered Configure link?

Doing this doesn't trigger addSupport() for me. Let me know if there's another way I'm overlooking.